packages/cli/test/fixtures/unit/commands/build/project-settings-override/package.json

```
@@ -0,0 +1,5 @@
+{
+  "scripts": {
+    "build": "this will be ignored because of the Project-level settings"
```

```suggestion
    "build": "mkdir -p output && echo 1 > output/index.txt"
```

How about a different value to prove it was overridden?